lib/Data/Xls.hs

```diff
@@ -179,15 +181,17 @@ decodeXlsByteString content = withSystemTempFile "decodeXlsByteString"
         hPut h content
         decodeXlsIO filePath
 
--- | Experimental: This function uses the @xls_open_buffer@ function of libxls.
-decodeXlsByteString' :: ByteString -> IO (Either XLSError [[[Cell]]])
+-- | Alternative to 'decodeXlsByteString': 
+-- This function uses the @xls_open_buffer@ function of libxls.
```

Comment:
This is hard to understand for users of the library. We can instead say something like:

"This function does not use a temporary file unlike decodeXlsByteString".

BTW, why don't we just use solely this API and remove the other one? Is this not reliable?